scipy/stats/_multivariate.py

```diff
@@ -3604,12 +3621,13 @@ def _to_corr(self, m):
         dimensionality. Note: modifies input matrix
         """
         # Check requirements for in-place Givens
-        if not (m.flags.c_contiguous and m.dtype == np.float64 and m.shape[0] == m.shape[1]):
+        if not (m.flags.c_contiguous and m.dtype == np.float64 and
+                m.shape[0] == m.shape[1]):
             raise ValueError()
```

Comment:
Perhaps adding a suitable error string for the `ValueError` might be helpful

Reply:
I agree, there *should* be a message there, but fixing that leads to another rabbit hole.  What should the message be?  Should there be a unit test for this code path?  I'll also leave this for another clean up, but we might as well ping the original author @philipdeboer and the committer @pv to see if they have any suggestions.